README.md

```diff
@@ -2,6 +2,94 @@
 
 This repository provides a GitHub Action for running the [Kani Rust Verifier](https://github.com/model-checking/kani) in CI.
 
+## Kani GitHub Action Parameters
+
+The following parameters can be used to configure and customize the behavior of this GitHub Action:
+
+**NOTE**: All the fields provided are optional and have default behaviors when not specified.
+
+`kani-version`
+
+- **Description**: Specifies the Kani version number to use.
```

Comment:
No need to say "Specifies" here nor in the other description. Number is just the way we represent versions, but that's not important. Here's my suggestion:
```suggestion
- **Description**: The Kani version to use.
```